content/en/examples/constants.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "constants"
+date: 2023-06-25T14:50:00-06:00
+draft: false
+---
+
+Constants store data that cannot be modified, just like immutable variables.
+
+Constants key differences are:
+- They are defined at compile time.
+- They are defined in the global scope.
+- Use the keyword `const` instead of `let`.
+- By convention, named in SCREAMING_SNAKE_CASE.
+- Their type must be annotated.
+- Currently they only can store literal constants
+
+```rust {.codebox}
+#[contract]
+mod contract_example {
+
+  const ONE_HOUR_IN_SECONDS: u32 = 3600;
+  const TICKTACKTOE_PLAYER_COUNT: u8 = 2;
+  const IS_COMPLETE: bool = 1;
+  ...
+  
```

Comment:
We should try to keep the examples as simple as possible. In this case, we can do that by:
- removing the `#[contract]` attribute (this can cause confusion)
- removing the module declarations and moving the consts outside of it (that way knowing about modules is not a prerequisite)

We could also add a `main` that uses the constants in some way (maybe prints/asserts some of them), to show how one would use them.